src/main/java/com/google/codeu/servlets/CreateNewEventServlet.java

```diff
@@ -75,12 +75,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     long timestamp = System.currentTimeMillis();
     String details = request.getParameter("details");
     String host = request.getParameter("host");
+    String lat = request.getParameter("lat");
+    String lng = request.getParameter("lng");
 
-    
-    Event event = new Event(user, title, date, time, timestamp, location, details, host);
-    // setEventImageUrl(request,event);
+    Event event = new Event(user, title, date, time, timestamp, location, details, host, lat, lng);
+    /*
```

Comment:
Somehow.../* doesn't comment out the setEventImageUrl

Reply:
it gets uncommented it the next pull request anyway :P